tdrs-backend/tdpservice/parsers/schema_defs/ssp/m4.py

```diff
@@ -0,0 +1,50 @@
+"""Schema for SSP M1 record type."""
+
+
+from tdpservice.parsers.util import SchemaManager
+from tdpservice.parsers.fields import Field
+from tdpservice.parsers.row_schema import RowSchema
+from tdpservice.parsers import validators
+from tdpservice.search_indexes.models.ssp import SSP_M4
+
+m4 = SchemaManager(
+      schemas=[
+        RowSchema(
+          model=SSP_M4,
+          preparsing_validators=[
+              validators.hasLength(66),
+          ],
+          postparsing_validators=[],
+          fields=[
+              Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
```

Comment:
please run Black on these, I had to run it to get linting to pass in #2734 . If we merge them like this, then we will get a lot of conflicts.